CafeRecords/Models/Owner.cs

```diff
@@ -1,16 +1,6 @@
-﻿// <auto-generated />
```

Comment:
Good idea to remove commented out code.  It's considered bad practice to commit commented-out code to version control, because you can always go back in history to see what used to be there.